package.json

```diff
@@ -32,10 +32,8 @@
     "es2015/text.js"
   ],
   "scripts": {
-    "build": "babel src -d .; NODE_ENV=es2015 babel src -d ./es2015; node script/inject-sequences.js",
-    "clean": "rm -rf -- index.js text.js es2015",
```

Comment:
`clean` still makes sense. The build script could be modified to stop generating one of the currently-generated files in the future.